django-stubs/db/models/fields/__init__.pyi

```diff
@@ -263,6 +264,31 @@ class TextField(Field[_ST, _GT]):
     _pyi_private_get_type: str
     # objects are converted to string before comparison
     _pyi_lookup_exact_type: Any
+    def __init__(
```

Comment:
Isn't it the same as in `CharField`?

Reply:
Thanks for the quick response!

Yes, the definition of `TextField.__init__()` is the same as that of `CharField`.

I'm not sure if I got your concern correctly, but if you are concerned about introducing unnecessary redundancy, I think we need to repeat the definition because `CharField` is not a subclass of `TextField` nor vice versa.

Reply:
👍